src/biogeochem/DryDepVelocity.F90

```diff
@@ -102,13 +102,15 @@ Module DryDepVelocity
   !------------------------------------------------------------------------
   subroutine Init(this, bounds)
 
-    use clm_varctl     , only : use_fates, use_fates_sp
+    use clm_varctl     , only : use_fates, use_fates_sp, use_fates_nocomp
     class(drydepvel_type) :: this
     type(bounds_type), intent(in) :: bounds
 
-    if ( (.not. use_fates_sp) .and. use_fates .and. (n_drydep > 0) ) then
-       call endrun( msg='ERROR: Dry-deposition currently does NOT work with FATES outside of FATES-SP mode (see github issue #1044)'//&
+    if (use_fates .and. (n_drydep > 0)) then    
+       if (( .not. use_fates_sp) .and. (.not. use_fates_nocomp)) then
```

Comment:
```suggestion
       if (.not. use_fates_nocomp) then
```

Reply:
Thanks. This makes things a lot easier. I was not sure if I could get rid of use_fates_sp there!